src/server/import-controller.js

```diff
@@ -60,6 +60,14 @@ class ImportController {
       res.send({ documents });
     });
 
+    router.get('/api/v1/imports/:batchId', [needsPermission(permissions.MANAGE_IMPORT)], async (req, res) => {
+      const { batchId } = req.params;
+      const rawBatch = await this.importService.getImportBatchDetails(batchId);
+      const batch = await this.clientDataMapper.mapImportBatch(rawBatch, req.user);
+
+      res.send({ batch });
+    });
+
     router.post('/api/v1/imports/batch', [jsonParser, needsPermission(permissions.MANAGE_IMPORT), validateBody(postImportBatchBodySchema)], async (req, res) => {
```

Comment:
Between this `POST` and the above `GET` we should have gone with
`POST /api/v1/imports/batches`
`GET /api/v1/imports/batches/:batchId`
🤦‍♀️  
Oh well, I guess it's too much of a hassle now to change and update both other websites for this.

Reply:
🤦 I could make this one  GET /api/v1/imports/batch/:batchId for consistency, but it would be correct to have "batches"

Reply:
We can do one further patch at the very end to just update to "batches" as a last thing in the epic. What do you think?

Reply:
Or then let's correct both endpoints and do another educandu update on the websites. We would need to do that anyway soon.

Reply:
Done, shall we make the updates?

Reply:
Um, that can only be done after we merge the PR that brings the update, right? 🙄 
Or did we misunderstand each other?

Reply:
Well, yes, after this thing is merged. But I think we don't need to rush it since it works even if the systems are running different versions of educandu. It's a breaking change only for the "current system" not for an external API